### PR TITLE
Fix kali tag selection

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Fix kali tag selection

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by TravisEz13** [comment]
@kiazhi

**Comment by TravisEz13** [comment]
@RDIL or @kiazhi  can your review?

File: .vsts-ci/phase.yml [hunk 1]
```diff
@@ -60,6 +60,15 @@ jobs:
       condition: succeededOrFailed()
       continueOnError: ${{ parameters.continueonerror }}
 
+  - powershell: |
+      $files = @(Get-ChildItem -Recurse test*.xml)
+      if($files.count -eq 0) {
+        throw 'no test results found'
+      }
+    displayName: Make sure we got test results
```
Comment (RDIL) [inline .vsts-ci/phase.yml]:
```suggestion
    displayName: Make sure we received test results
```

File: release/community-stable/parrot/docker/Dockerfile [hunk 2]
```diff
@@ -58,6 +58,7 @@ RUN \
       locales \
       # required to support NTLM for PSRemoting
       gss-ntlmssp \
+      resolvconf \
```
Comment (RDIL) [inline release/community-stable/parrot/docker/Dockerfile]: What is this for?

File: release/community-stable/parrot/docker/Dockerfile [hunk 3]
```diff
@@ -66,8 +67,15 @@ RUN \
...
+    # upgrade distro
+    && apt-get dist-upgrade -y \
+    # clean downloaded packages
+    && apt-get clean \
+    # remove package list
+    && rm -rf /var/lib/apt/lists/*
+
     # install powershell package
-    && apt-get install -y /tmp/powershell.deb \
+RUN apt-get install -y /tmp/powershell.deb \
```
Comment (RDIL) [inline release/community-stable/parrot/docker/Dockerfile]: I feel like this extra layer really isn't needed

File: release/community-stable/parrot/docker/Dockerfile (hunk 2 above)
Comment (TravisEz13) [reply release/community-stable/parrot/docker/Dockerfile]: required for the distupgrade

File: release/community-stable/parrot/docker/Dockerfile (hunk 3 above)
Comment (TravisEz13) [reply release/community-stable/parrot/docker/Dockerfile]: I added it during debugging for faster perf.  I will remove.

File: .vsts-ci/phase.yml [hunk 4]
```diff
@@ -60,6 +60,15 @@ jobs:
       condition: succeededOrFailed()
       continueOnError: ${{ parameters.continueonerror }}
 
+  - powershell: |
```
Comment (RDIL) [inline .vsts-ci/phase.yml]: To be completely honest, to this day, Azure DevOps has never made sense to me.  These config files just ramble on, and its so confusing.  I really wish that powershell could use a different CI.  Just saying.

Comment (TravisEz13) [reply .vsts-ci/phase.yml]: It's very flexible...
I love it.

Comment (TravisEz13) [reply .vsts-ci/phase.yml]: And it's Microsoft... So, I think you know the likelihood of us moving to something else.

Comment (RDIL) [reply .vsts-ci/phase.yml]: I can kinda see what you mean, but I just feel like all these different aspects could be improved a ton.  The docs also really don't help either.

Comment (TravisEz13) [reply .vsts-ci/phase.yml]: The language, probably could be a bit simpler and accomplish the same thing, but they had an existing GUI based system that they tried to fit into a YAML which somewhat limited their options.

Completely agree about the docs.

Comment (RDIL) [reply .vsts-ci/phase.yml]: Ouch, that's an awful port, GUI to YAML 👀 

Also, maybe they should just think about put the docs in here:

Comment (TravisEz13) [reply .vsts-ci/phase.yml]: They at least need a major restructuring.

Comment (RDIL) [reply .vsts-ci/phase.yml]: Agreed

**Pull request merged**